data/crac-io/crac-io-json/src/main/java/com/powsybl/openrao/data/craciojson/JsonImport.java

```diff
@@ -29,41 +29,57 @@
  * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
  * @author Peter Mitri {@literal <peter.mitri at rte-france.com>}
  */
-@AutoService(CracImporter.class)
-public class JsonImport implements CracImporter {
-    private static final String JSON_EXTENSION = "json";
+@AutoService(Importer.class)
+public class JsonImport implements Importer {
+    private byte[] data;
 
-    private Crac importCrac(InputStream inputStream, CracDeserializer cracDeserializer) {
+    public JsonImport() {
+        data = null;
+    }
+
+    @Override
+    public String getFormat() {
+        return "JSON";
+    }
+
+    @Override
+    public boolean exists(InputStream inputStream) {
         try {
```

Comment:
use CracDeserializer.checkHeader instead of try/catch. rename checkHeader to isValid